nixos/modules/services/web-apps/phylactery.nix

```diff
@@ -0,0 +1,44 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let cfg = config.services.phylactery;
+in {
+  options.services.phylactery = {
+    enable = mkEnableOption "Whether to enable Phylactery server.";
+
+    host = mkOption {
+      type = types.str;
+      default = "";
+      description = "Listen host for Phylactery.";
+    };
+
+    port = mkOption {
+      type = types.port;
+      description = "Listen port for Phylactery.";
+    };
+
+    library = mkOption {
+      type = types.str;
+      description = "Path to CBZ library.";
+    };
+  };
+
+  config = lib.mkIf cfg.enable {
+    systemd.services.phylactery = {
+      environment = {
+        PHYLACTERY_ADDRESS = "${cfg.host}:${toString cfg.port}";
+        PHYLACTERY_LIBRARY = cfg.library;
+      };
+
+      wantedBy = [ "multi-user.target" ];
+
+      serviceConfig = {
+        ConditionPathExists = cfg.library;
```

Comment:
What does it do if it isn't around?

Reply:
Everything would respond as 404 so this check is nice to have IMHO.